forecasttools/trajectories_to_quantiles.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+import polars as pl
+
+
+def trajectories_to_quantiles(
+    trajectories: pl.DataFrame,
+    quantiles: list[int] = None,
+    timepoint_cols: list[str] = ["timepoint"],
+    value_col: str = "value",
+    quantile_value_name: str = "quantile_value",
+    quantile_level_name: str = "quantile_level",
+    id_cols: list[str] = None,
+) -> pl.DataFrame:
+    # set default quantiles
+    if quantiles is None:
+        quantiles = (
+            [0.01, 0.025] + list(np.arange(0.05, 1.0, 0.05)) + [0.975, 0.99]
+        )
+    # group trajectories based on timepoint_cols and id_cols
+    group_cols = (
+        timepoint_cols if id_cols is None else timepoint_cols + id_cols
+    )
```

Comment:
Seems like `timepoint_cols` and id_cols are only used to define `group_cols`. Why not just take `group_cols` as the argument?

Reply:
Will do. Thank you.